Purchases/Caching/DeviceCache.swift

```diff
@@ -334,8 +334,8 @@ class DeviceCache {
     fileprivate struct CacheKeyBases {
 
         static let keyBase = "com.revenuecat.userdefaults."
-        static let purchaserInfoAppUserDefaults = "\(keyBase)purchaserInfo."
```

Comment:
I honestly wouldn't mind having the data migration, so that we don't references to `customerInfo`, `purchaserInfo` and `subscriberInfo` in the same codebase at the same time, but we can tackle it later

Reply:
Same, it's just kind of a big enough deal that I'd want this to be a separate, well thought out thing.